crmsh/constants.py

```diff
@@ -454,4 +454,9 @@
 
 # Commands that are deprecated and hidden from UI
 HIDDEN_COMMANDS = {'ms'}
+
+# pacemaker crm/common/results.h
```

Comment:
Just hit another one:
```
      Stderr: �[33mWARNING�[0m: Unknown return code from crmadmin: 113
      �[33mWARNING�[0m: DC lost during wait
```
which is  
`CRM_EX_UNSATISFIED          = 113, //!< Requested item does not satisfy constraints`
and it seems like this case mapped on `./lib/common/results.c` in pacemaker:
```
        case EAGAIN:
        case EBUSY:
            return CRM_EX_UNSATISFIED;
```
So, worth to ignore that RC and give another try.

Reply:
So, just reusing the original version of `get_dc`, ignoring the exit status, is good enough.